third_party/ssc/jsoncpp/CMakeLists.txt

```diff
@@ -142,16 +142,16 @@ endif()
 if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
     # using regular Clang or AppleClang
     if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.12.0)
-      add_compile_options(-Wall -Wconversion -Wshadow -Werror=conversion -Werror=sign-compare)
+      add_compile_options(-Wall -Wconversion -Wshadow -Werror=conversion -Werror=sign-compare -Wno-deprecated-declarations)
```

Comment:
Avoid the annoying "Reader is deprecated" warning that you get in jsoncpp (deps of ssc)

Reply:
I hadn't paid attention or seen that warning, but OK.  Is there anything else we should do for this issue long term?